pkg/shared/remote/DynamicServiceConstants.go

```diff
@@ -19,4 +19,5 @@ type AmsConfig struct {
 	TrialProductQuotaID         string `json:"trialQuotaProductId"`
 	LongLivedQuotaProductID     string `json:"longLivedQuotaProductId"`
 	ResourceName                string `json:"resourceName"`
+	EnterpriseProductQuotaID    string `json:"RHOSAKCC"`
```

Comment:
```suggestion
	EnterpriseProductQuotaID    string `json:"enterpriseProductQuotaID"`
```

Reply:
This also needs to be added here https://github.com/redhat-developer/app-services-cli/blob/b5cbad899af6c9ae215a2401f463888747039b16/pkg/shared/remote/service-constants.json#L4

Reply:
got this on the other PR will close this one. thanks for the great catch 👍